src/devices/ecowitt.c

```diff
@@ -0,0 +1,115 @@
+/*
+ * 55-bit one-row data packet format (inclusive ranges):
+ * |  0-14 | Always 111111101010011
```

Comment:
This matches this code:

    } else if (bitbuffer->bits_per_row[0] == 55 &&
            bb[0][0] == 0xFE) { // WH2A
        bitbuffer_extract_bytes(bitbuffer, 0, 7, b, 48);
        model = _X("Fineoffset-WH2A","Fine Offset WH2A sensor");

Maybe the crc check rejects the data.

Reply:
Yeah, looks very similar. I wonder if we could merge these, but my guess is perhaps not: They're slightly different: the WH2A has a 4-byte checksum, but this has a 5-byte checksum.